gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/GqlQuery.java

```diff
@@ -373,26 +357,29 @@ public boolean equals(Object obj) {
   }
 
   @Override
-  protected DatastoreV1.GqlQuery toPb() {
-    DatastoreV1.GqlQuery.Builder queryPb = DatastoreV1.GqlQuery.newBuilder();
+  protected com.google.datastore.v1beta3.GqlQuery toPb() {
+    com.google.datastore.v1beta3.GqlQuery.Builder queryPb = 
+        com.google.datastore.v1beta3.GqlQuery.newBuilder();
     queryPb.setQueryString(queryString);
-    queryPb.setAllowLiteral(allowLiteral);
-    for (Binding argument : namedBindings) {
-      queryPb.addNameArg(argument.toPb());
+    queryPb.setAllowLiterals(allowLiteral);
+    Map<String, com.google.datastore.v1beta3.GqlQueryParameter> namedBindingsPb = 
+        queryPb.getMutableNamedBindings();
+    for (Map.Entry<String, Binding> entry : namedBindings.entrySet()) {
+      namedBindingsPb.put(entry.getKey(), entry.getValue().toPb());
     }
     for (Binding argument : positionalBindings) {
-      queryPb.addNumberArg(argument.toPb());
+      queryPb.addPositionalBindings(argument.toPb());
     }
     return queryPb.build();
   }
 
   @Override
-  protected void populatePb(DatastoreV1.RunQueryRequest.Builder requestPb) {
+  protected void populatePb(com.google.datastore.v1beta3.RunQueryRequest.Builder requestPb) {
     requestPb.setGqlQuery(toPb());
   }
 
   @Override
-  protected GqlQuery<V> nextQuery(DatastoreV1.QueryResultBatch responsePb) {
+  protected GqlQuery<V> nextQuery(com.google.datastore.v1beta3.QueryResultBatch responsePb) {
```

Comment:
We can implement it now, but also fine if we do that after we merge to main.
I assigned the issue (#17) to you.